std/traits.d

```diff
@@ -77,6 +77,7 @@
  *           $(LREF StringTypeOf)
  *           $(LREF AssocArrayTypeOf)
  *           $(LREF BuiltinTypeOf)
+ *           $(LREF TypeOf)
```

Comment:
That wouldn't match the naming conventions or how other templates in std.traits are named.